draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -566,15 +577,16 @@ subflows.
 
 
 ~~~~
-  +--------+--------+--------+--------+--------+--------+--------+
-  |00101110| Length |00000011|Key Type| Key Data ...
-  +--------+--------+--------+--------+--------+--------+--------+
+  +--------+--------+--------+-----------+-------------+-----------+-------------+-----
+  |00101110| Length |00000011|Key Type(1)| Key Data(1) |Key Type(2)| Key Data(2) | ....
+  +--------+--------+--------+-----------+-------------+-----------+-------------+-----
    Type=46           MP_OPT=3
 ~~~~
 
 The MP_KEY suboption is used to exchange key material between
-hosts. The Length varies between 5 and 8 Bytes.  The Key Type field is used to
-specify the key type. Key types are shown in {{ref-key-type-list}}.
+hosts. The Length varies between 12 and 36 Bytes for a single-key message, and up to
+78 bytes when multiple keys are provided. The Key Type field is used to
+specify the type of the following key data. Key types are shown in {{ref-key-type-list}}.
 
 | Key  Type              | Key Length | Meaning                          |
```

Comment:
Do we have to specify, that Key Length defines Bytes here?

Reply:
Better to add it to avoid confusion. I've also noticed that key type ECDHE-C25519-SHA512 has length 32, but a SHA512 hash is 64 bytes: is it a error in the length or do we expect to truncate it to 32 bytes?

Reply:
Then I propose to adapt to 64 Bytes. Please consider the impact this has on the maximum single-key and multi-key length